src/pages/forms/createAccount.js

```diff
@@ -0,0 +1,312 @@
+import React, { useEffect, useRef, useState } from "react";
+import styles from "./createAccount.module.css";
+import { Link } from "react-router-dom";
+import { useParams } from "react-router-dom";
+const CreateAccount = () => {
+  const [showEye, setShowEye] = useState(true);
+  const [showPass, setShowPass] = useState(true);
+  const [radioMarked, setRadioMarked] = useState(false);
+  const [randomVal1, setRandomVal1] = useState(true);
+  const [randomVal2, setRandomVal2] = useState(true);
+  const [showFirstName, setShowFirstName] = useState(false);
```

Comment:
@HashIqbal  what are these values being used for showFirstName etc .  check how to handle form values .
Dont use extra states , code should be refactored.